test/test_prototype_transforms_consistency.py

```diff
@@ -254,9 +254,10 @@ def __init__(
         legacy_transforms.RandomAdjustSharpness,
         [
             ArgsKwargs(p=0, sharpness_factor=0.5),
-            ArgsKwargs(p=1, sharpness_factor=0.3),
+            ArgsKwargs(p=1, sharpness_factor=0.2),
```

Comment:
Why is this needed? Don't we tolerance for that?

Reply:
This helps to fix tolerance without adjusting a/r tol :)

Reply:
In that case, why do we need it do it 3 lines below?

Reply:
there are float tol and uint8 tol. `sharpness_factor=0.2` avoids uint8 tol issue. `closeness_kwargs={"atol": 1e-6, "rtol": 1e-6},` helps with float tol issue